packages/@aws-cdk/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -420,6 +529,14 @@ export interface FlowLogOptions {
    */
   readonly destination?: FlowLogDestination;
 
+  /**
+   * The fields to include in the flow log record, in the order in which they should appear.
```

Comment:
If you specify a custom log format do you now have to specify any of the default fields that you
want to include? For example if you want to include all default fields + a single custom field do
you have to specify _all_ the fields? If so, it might be nice to add an option to `LogFormatField`
to include all the default fields (i.e. `LogFormatField.ALL_DEFAULT_FIELDS`.

Reply:
I thought that was the best name, so I used it as a reference.

Reply:
Add something to the effect of:

```suggestion
   * The fields to include in the flow log record, in the order in which they should appear.
   * 
   * If multiple fields are specified, they will be separated by spaces. For full control over the literal log format
   * string, pass a single field constructed with `LogFormat.custom()`.
   *
```